src/VoteDelegate.t.sol

```diff
@@ -193,14 +197,59 @@ contract VoteDelegateTest is DSTest {
 
         // Comply with Chief's flash loan protection
         hevm.roll(block.number + 1);
+        hevm.warp(block.timestamp + 1);
+
+        delegate.doProxyFree(100 ether);
+        assertEq(gov.balanceOf(address(delegate)), 100 ether);
+        assertEq(gov.balanceOf(address(chief)), currMKR);
+        assertEq(iou.balanceOf(address(delegate)), 0);
+        assertEq(proxy.stake(address(delegate)), 0);
+    }
+
+    function test_proxy_lock_free_after_expiration() public {
+        uint256 currMKR = gov.balanceOf(address(chief));
+
+        delegate.approveGov(address(proxy));
+        delegate.approveIou(address(proxy));
+
+        assertEq(gov.balanceOf(address(delegate)), 100 ether);
+        assertEq(iou.balanceOf(address(delegate)), 0);
+
+        delegate.doProxyLock(100 ether);
+        assertEq(gov.balanceOf(address(delegate)), 0);
+        assertEq(gov.balanceOf(address(chief)), currMKR + 100 ether);
+        assertEq(iou.balanceOf(address(delegate)), 100 ether);
+        assertEq(proxy.stake(address(delegate)), 100 ether);
+
+        // Flash loan protection
+        hevm.roll(block.number + 1);
 
+        // Warp past expiration
+        hevm.warp(block.timestamp + 9001 days);
```

Comment:
File-level comment here: it would be nice to have at least one test that probes the exact magnitude of the expiration, i.e. it `warps` to just before expiry and confirms that everything works, then `warps` to just after expiry and confirms things have been shut down. We can keep 9001 everywhere else, ofc.

Reply:
Added.